core/computed/entity-classification.js

```diff
@@ -13,16 +13,51 @@ import thirdPartyWeb from '../lib/third-party-web.js';
 /** @typedef {Map<string, LH.Artifacts.Entity>} EntityCache */
 
 class EntityClassification {
+  /**
+   * @param {EntityCache} entityCache
+   * @param {string} url
+   * @param {string=} optionalName
+   * @return {LH.Artifacts.Entity}
+   */
+  static makupChromeExtensionEntity(entityCache, url, optionalName) {
+    const origin = Util.getChromeExtensionOrigin(url);
+    const host = new URL(origin).host;
+    const name = optionalName || host;
```

Comment:
I'm defaulting unknown chrome extensions' names to the `host` part of the extension origin URL, i.e., `foobar` in `chrome-extension://foobar/baz.js`. This should help searching on the web with that string to reach the extension.